src/fluree/db/api.cljc

```diff
@@ -289,7 +288,7 @@
   ([db policy policy-values]
    (promise-wrap
     (let [policy* (json-ld/expand policy)]
-      (policy/wrap-policy db policy* policy-values)))))
+      (policy/wrap-policy db nil policy* policy-values)))))
```

Comment:
we should change `wrap-policy` (and any other function that takes optional arguments) into a multi-arity function so we don't have to pass `nil` like this.